### PR TITLE
Wiki: add guide for updating

### PR DESCRIPTION
**Pull request by extremepayne**
I'm not particularly attached to any of what I wrote; feel free to edit it. But I felt this was an obvious topic to have a wiki article for.

### CODE REVIEW

File: src/wiki/getting-started/updating-prismlauncher.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  order: 12
+---
+
+# Updating Prism Launcher
+
+Updating Prism Launcher will depend a little on the method of installation you chose.
+
+## Any kind of package management system
+
+This includes Scoop, Chocolatey, Winget, Homebrew, MacPorts, FlatHub, `apk`, any aur helper, `dnf`, `apt`, any MPR helper, Pacstall, the pi-apps store, `emerge`, `zypper`, and `xbps`.
```
Comment (RaptaG) [inline src/wiki/getting-started/updating-prismlauncher.md:14]: AUR needs capital letters

Comment (extremepayne) [reply src/wiki/getting-started/updating-prismlauncher.md]: addressed in 02f9ed5

**Review by RaptaG — APPROVED** [review]

**Comment by extremepayne** [comment]
the CI failure is due to a rate limiting issue, not anything with my PR

if there's anything I can do to help this go through lmk

File: src/wiki/getting-started/updating-prismlauncher.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  order: 12
+---
+
+# Updating Prism Launcher
+
+Updating Prism Launcher will depend a little on the method of installation you chose.
+
+## Any kind of package management system
+
+This includes Scoop, Chocolatey, Winget, Homebrew, MacPorts, FlatHub, `apk`, any AUR helper, `dnf`, `apt`, any MPR helper, Pacstall, the pi-apps store, `emerge`, `zypper`, and `xbps`.
```
Comment (Scrumplex) [inline src/wiki/getting-started/updating-prismlauncher.md:14]:
```suggestion
This includes Scoop, Chocolatey, winget, Homebrew, MacPorts, Flathub, `apk`, any AUR helper, `dnf`, `apt`, any MPR helper, Pacstall, the Pi-Apps store, `emerge`, `zypper`, and `xbps`.
```

**Comment by extremepayne** [comment]
The original reason I wrote this was to clarify the somewhat archaic Windows process as I saw a lot of questions on the discord about Windows updates and a very formulaic response that I felt would be helped by having an FAQ article that explains the process and conservation of instances. 

If we have auto-update on Windows then I don't see a significant reason for this to exist, but also wouldn't be opposed to it existing to just explain which ones auto-update and which ones need manual updating with a package manager. Basically I could see it go either way.

**Comment by extremepayne** [comment]
Yeah, there's no need

**Pull request closed without merging**